src/Npc.js

```diff
@@ -99,6 +99,27 @@ class Npc extends Character {
       this.addItem(newItem);
     });
 
+    // Hydrate equipment
+    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
+      const eqDefs = this.defaultEquipment;
+      this.defaultEquipment = new Map();
+      for (const slot in eqDefs) {
+        const itemDef = eqDefs[slot];
+        try {
+          let newItem = state.ItemFactory.create(state.AreaManager.getAreaByReference(itemDef), itemDef);
+          newItem.initializeInventory(itemDef.inventory);
+          newItem.hydrate(state);
+          state.ItemManager.add(newItem);
+          newItem.isEquipped = true;
+          this.equip(newItem, newItem.metadata.slot);
+        } catch (e) {
+          Logger.error(e.message);
+        }
+      }
+    } else {
+      this.defaultEquipment = new Map();
```

Comment:
`defaultEquipment` should be `equipment`. `defaultEquipment` should be the _original_ value of `equipment` from the YAML file. The purpose being that an NPC's equipment may change over time by equipping/unequipping so to access what their starting equipment should be there is `defaultEquipment`

Reply:
on line 120 I take it you're referring to?